purview/models/errata.py

```diff
@@ -28,15 +29,13 @@ class Advisory(PurviewStructuredNode):
     type_ = StringProperty(db_property='type')
     update_date = DateTimeProperty()
     updated_at = DateTimeProperty()
-    assigned_to = RelationshipTo('.user.User', 'ASSIGNED_TO')
-    attached_bugs = RelationshipTo('.bugzilla.BugzillaBug', 'RELATED_TO')
-    attached_builds = RelationshipTo('.koji.KojiBuild', 'RELATED_TO')
-    # A normalized relationship name (same as reporters)
-    owners = RelationshipTo('.user.User', 'OWNED_BY')
-    package_owners = RelationshipTo('.user.User', 'PACKAGE_OWNED_BY')
-    reporters = RelationshipTo('.user.User', 'REPORTED_BY')
-    states = RelationshipTo('AdvisoryState', 'RELATED_TO')
-    triggers_freshmaker_event = RelationshipTo('.freshmaker.FreshmakerEvent', 'TRIGGERS')
+    assigned_to = RelationshipTo('.user.User', 'ASSIGNED_TO', cardinality=ZeroOrOne)
+    attached_bugs = RelationshipTo('.bugzilla.BugzillaBug', 'ATTACHED')
+    attached_builds = RelationshipTo('.koji.KojiBuild', 'ATTACHED')
```

Comment:
Again, according to the story flow, I was wondering if we could have RelationshipFrom here and RelationshipTo in the KojiBuild model